include/openssl/core_names.h.in

```diff
@@ -66,6 +66,9 @@ extern "C" {
 
 /* Known KDF names */
 # define OSSL_KDF_NAME_HKDF           "HKDF"
+# define OSSL_KDF_NAME_HKDF_SHA256    "HKDF-SHA256"
```

Comment:
Is this naming okay? Or "HKDF-with-SHA256"? Or something else?

Reply:
name is fine, we dont do with elsewhere in the providers.